content/_data/logo/alien.yml

```diff
@@ -0,0 +1,9 @@
+---
+name: 'Alien'
+url: '/images/logos/alien/alien.png'
+url_256: '/images/logos/alien/256.png'
+vector:
+credit: 'Kseniia Nenasheva'
+credit_url: 'https://twitter.com/ks_nenasheva'
```

Comment:
@ksenia-nenasheva Are you fine with the twitter reference. Or do you prefer a GitHub one? Also "Ksenia" or "Kseniia"?